test/serializers/default_include_test.rb

```diff
@@ -0,0 +1,52 @@
+require 'test_helper'
+
+module ActiveModel
+  class Serializer
```

Comment:
Sorry not sure I got this one. I think this is what I did.

> Can you add a test for if a post has default_includes, but the object you're serializing does not?

`default_include` is specified in the PostSerializer, but in the test I'm serialising a Blog object.